doc/policy/README.md

```diff
@@ -0,0 +1,12 @@
+# Mempool Policy
+
+**Mempool Policy** (aka Transaction Relay Policy) is the node's set of validation rules, in addition
+to consensus, enforced for unconfirmed transactions before submitting them to the mempool.  These
+local rules include restrictions on the transaction itself and the transaction in relation to the
+node's mempool contents. They are *not* applied to transactions in blocks.
+
```

Comment:
Maybe it should precise that a subset of those rules are fine-tunable by the node operator (`-mintxfee`, `-limitdescendantsize`, `-limitancestorsize`, ...) ?

I think it's interesting to notice that, to underscore that those rules might not even be uniform across the Bitcoin Core node of the same version.

Reply:
Ah true, added " These rules are local to the node and configurable (e.g. `-minrelaytxfee`, `-limitancestorsize`,`-incrementalRelayFee`)."